### PR TITLE
Update Container image

### PR DESCRIPTION
**Pull request by rwaffen**

### CODE REVIEW

**Comment by rwaffen** [comment]
see #48

File: Dockerfile [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ARG UBUNTU_CODENAME=jammy
 
-FROM ubuntu:${UBUNTU_CODENAME}
+FROM ghcr.io/betadots/ruby:3.3.0-${UBUNTU_CODENAME}
```
Comment (bastelfreak) [inline Dockerfile:3]: Is it sane to always use this version when we do rspec-puppet testing? That will load the puppet 7 gem on ruby 3.3 which isn't officially tested. I don't mind a newer version but I don't know if it works.

Comment (rwaffen) [reply Dockerfile:3]: good point. didn't thought about that 🤔. this sh** is getting more and more complicated. i just wanted to have an easy and simple container :-D

Comment (rwaffen) [reply Dockerfile:3]: Huh. This is getting more complicated. Ubuntu jammy has only ruby 3.0 as system ruby. The dockerfile from ruby for 2.7 only build with Ubuntu focal. 

It's a mess. ¯\_(ツ)_/¯

Comment (bastelfreak) [reply Dockerfile:3]: Is it an option to use the ruby 2.7 Buster and ruby 3.2 Buster container images as base, until Puppet publishes bolt/pdk packages for Debian Bookworm?

Comment (rwaffen) [reply Dockerfile:3]: Ah okay. They only deleted the Dockerfiles. But the containers are still available.

Comment (bastelfreak) [reply Dockerfile:3]: Hrmpf yes it's really a challenge that LTS puppet relies on dead upstream software. I raised concerns about this a few times but without much success.

Comment (rwaffen) [reply Dockerfile]: i added the base image as an parameter and try now to build upon ruby:$VERSION-bullseye

Comment (rwaffen) [reply Dockerfile]:
```
54.34 ERROR:  Error installing bundler:
54.34 	There are no versions of bundler (= 2.5.6) compatible with your Ruby & RubyGems
54.34 	bundler requires Ruby version >= 3.0.0. The current ruby version is 2.7.8.225.
```

Comment (rwaffen) [reply Dockerfile]: Okay now searching an older bundle for the older ruby

**Pull request closed without merging**